optuna/importance/_fanova/_fanova.py

```diff
@@ -80,51 +79,35 @@ def fit(
             # This could occur if for instance `X.shape[0] == 1`.
             raise RuntimeError("Encountered zero total variance in all trees.")
 
-    def get_importance(self, features: Tuple[int, ...]) -> Tuple[float, float]:
+    def get_importance(self, feature: int) -> Tuple[float, float]:
```

Comment:
Perhaps we should update the documentation at the top of this file that mentions pairwise importances (which are no longer computable after this change).

Reply:
+1. In addition, we need to change the documentation of `FanovaImportanceEvaluator` https://github.com/optuna/optuna/blob/master/optuna/importance/_fanova/_evaluator.py#L36.

Reply:
Thanks! Good catch. I'll update the documentation.